src/main/java/org/cryptomator/common/settings/SettingsJsonAdapter.java

```diff
@@ -82,7 +82,7 @@ public Settings read(JsonReader in) throws IOException {
 				case "preferredVolumeImpl" -> settings.preferredVolumeImpl().set(parsePreferredVolumeImplName(in.nextString()));
 				case "theme" -> settings.theme().set(parseUiTheme(in.nextString()));
 				case "uiOrientation" -> settings.userInterfaceOrientation().set(parseUiOrientation(in.nextString()));
-				case "keychainBackend" -> settings.keychainBackend().set(parseKeychainBackend(in.nextString()));
```

Comment:
By removing/renaming the property on reading the settings file, it can break existing setups.

We should keep it for now and add a comment to remove it in a future release.

Reply:
Ok, update: This is a deliberate choice. 
We reduce legacy code and possible invalid values here, because we need actual class names, which the former enum values weren't.

Only KWallet user would be affected with the consequence that no Keychain is selected. It can be easily und quickly fixed by reselecting the KWallet Keychain.